geometry_msgs/Point32 position
geometry_msgs/Point32 direction

bool is_visual
